Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>**Welcome to my first Java Project**\r\n\r\nThis GUI application allow users to track their travel records and update their bucket list!  \r\n\r\nThis project was developed with the help from the entire instruction team of CPSC 210 at UBC and consisted of four phases:\r\n1. Model construction and commend line user interaction\r\n2. Data persistence \r\n3. Graphical User Interface \r\n4. Design decision   \r\n\r\nCode were tested during the implementation, and all changes are committed regularly.   \r\nI tried my best writing this program; however, I also acknowledge that the final GUI is very redundant and there are plenty room for improvement. Stay tuned for updates on version 2! 
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	
+++ b/README.md	
@@ -8,5 +8,6 @@
 3. Graphical User Interface 
 4. Design decision   
 
+!()
 Code were tested during the implementation, and all changes are committed regularly.   
-I tried my best writing this program; however, I also acknowledge that the final GUI is very redundant and there are plenty room for improvement. Stay tuned for updates on version 2! 
\ No newline at end of file
+I tried my best writing this program; however, I also acknowledge that there are redundant code in MainGUI and there are plenty room for improvement. Stay tuned for updates on version 2! 
\ No newline at end of file
